webapp/app/js/compareperf.js

```diff
@@ -255,6 +278,16 @@ perf.controller('CompareSubtestResultsCtrl', [
       });
     }
 
+    $scope.dataLoading = true;
+    $scope.getDeltaClass = function(cr) { return cr.className; };
```

Comment:
Code duplication with the above. Maybe we should put the function I suggested above in the compare factory and then make the $scope method a simple wrapper.